core/services/functions/request.go

```diff
@@ -18,6 +18,21 @@ type RequestData struct {
 	Args            []string `json:"args" cbor:"args"`
 }
 
+type SignedRequestDataV1 struct {
+	CodeLocation    int    `json:"codeLocation" cbor:"codeLocation"`
+	Language        int    `json:"language" cbor:"language"`
+	Secrets         []byte `json:"secrets" cbor:"secrets"`
+	SecretsLocation int    `json:"secretsLocation" cbor:"secretsLocation"`
+	Source          string `json:"source" cbor:"source"`
+}
+
+type RequestDataV1 struct {
```

Comment:
I think it will work if you add all params to ReqeustData. In v0 some of them will just be empty. Can you check?